app/ui.js

```diff
@@ -383,7 +381,7 @@ var UI = {
         document.documentElement.classList.remove("noVNC_disconnecting");
         document.documentElement.classList.remove("noVNC_reconnecting");
 
-        let transition_elem = document.getElementById("noVNC_transition_text");
+        const transition_elem = document.getElementById("noVNC_transition_text");
```

Comment:
There's a whole bunch of these unnecessary changes from `let` to `const`.

Reply:
I thought that the let/const commit was approved already... I haven't touched that

We already discussed the let vs const stuff above. The const enforcing is removed.

Do you want me now to go back and change the const to let?